fw/tls.c

```diff
@@ -100,19 +101,24 @@ tfw_tls_connection_recv(TfwConn *conn, struct sk_buff *skb)
 		BUG_ON(r == T_BLOCK);
 		fallthrough;
 	case T_BAD:
-		r = T_BAD;
 		if (tls->conf->endpoint == TTLS_IS_SERVER)
 			TFW_INC_STAT_BH(serv.tls_hs_failed);
 		fallthrough;
 	case T_BLOCK_WITH_FIN:
+		r = T_BLOCK_WITH_FIN;
```

Comment:
Why do we assign `T_BLOCK_WITH_FIN` to `r` here?

Reply:
Yes fixed